Makefile

```diff
@@ -20,6 +20,7 @@ test_all: test_results
 	@echo "+++ testing - all the tests"
 	@echo
 	$(call GOTESTCMD,$@) -tags all --timeout 60s -v ./...
+	cd tests && $(call GOTESTCMD,$@) -tags all --timeout 60s -v ./...
```

Comment:
Can you break this out into its own make command and then use it here

Reply:
sure.